variables.ini.dist

```diff
@@ -67,14 +67,14 @@ AUTH_ENABLE_OPENSTACK =
 KEYSTONE_TOKEN_LOGIN = # True to use Keystone tokens instead of passwords.
 KEYSTONE_SERVER = # "https://my-openstack.cloud.org:5000"
 KEYSTONE_DOMAIN_NAME = # "default"
-MOCK_USER =
+MOCK_USER = mock_user
 USE_MOCK_DATA = False # True to use fake api when possible
 USE_ALLOCATION_SOURCES = True # True for jetstream
-DATABASE_NAME = troposphere_db
+DATABASE_NAME = troposphere
```

Comment:
I learned that last night, when Travis went 💥 . I switched it back to `troposphere_db `.